{"name": "Email4 -- @1.1 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: target frame detached\n  (Session info: chrome=87.0.4280.66)\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/basic_steps.py\", line 88, in step_impl\n    context.current_page.get_element(element_name, timeout=1)\n  File \"/Users/kardash/Developer/fitness-automation/features/pages/base_page.py\", line 49, in get_element\n    return WebDriverWait(self.driver, timeout).until(\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/wait.py\", line 71, in until\n    value = method(self._driver)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/expected_conditions.py\", line 64, in __call__\n    return _find_element(driver, self.locator)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/expected_conditions.py\", line 415, in _find_element\n    raise e\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/expected_conditions.py\", line 411, in _find_element\n    return driver.find_element(*by)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 976, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given clear time", "status": "passed", "start": 1608418830278, "stop": 1608418830278}, {"name": "When open home page", "status": "passed", "start": 1608418830279, "stop": 1608418833566}, {"name": "When calculate time for 'home'", "status": "passed", "start": 1608418833567, "stop": 1608418833567}, {"name": "When go to confirmation page", "status": "passed", "start": 1608418833567, "stop": 1608418849710}, {"name": "When calculate time for 'confirmation'", "status": "passed", "start": 1608418849710, "stop": 1608418849710}, {"name": "When go to cloudpayments page", "status": "passed", "start": 1608418849711, "stop": 1608418857766}, {"name": "When calculate time for 'payment'", "status": "passed", "start": 1608418857766, "stop": 1608418857767}, {"name": "When enter fake card details", "status": "passed", "start": 1608418857767, "stop": 1608418862536}, {"name": "Then I see 3d secure element", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: target frame detached\n  (Session info: chrome=87.0.4280.66)\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/basic_steps.py\", line 88, in step_impl\n    context.current_page.get_element(element_name, timeout=1)\n  File \"/Users/kardash/Developer/fitness-automation/features/pages/base_page.py\", line 49, in get_element\n    return WebDriverWait(self.driver, timeout).until(\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/wait.py\", line 71, in until\n    value = method(self._driver)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/expected_conditions.py\", line 64, in __call__\n    return _find_element(driver, self.locator)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/expected_conditions.py\", line 415, in _find_element\n    raise e\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/expected_conditions.py\", line 411, in _find_element\n    return driver.find_element(*by)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 976, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "attachments": [{"name": "bug.png", "source": "7e3eb807-00e0-4eb2-b950-795cce1b40e9-attachment.png", "type": "image/png"}], "start": 1608418862536, "stop": 1608418870329}, {"name": "When calculate time for 'purchace'", "status": "skipped", "start": 1608418870329, "stop": 1608418870329}, {"name": "When wait for \"1\" sec", "status": "skipped", "start": 1608418870329, "stop": 1608418870329}], "parameters": [{"name": "card_type", "value": "fake card"}], "start": 1608418830276, "stop": 1608418870329, "uuid": "c48c4b18-02c5-4f38-b31c-470ffacca70b", "historyId": "e0df3e86b22d6b65aeab1d3b170652d5", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "validation"}, {"name": "feature", "value": "Purchase"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}